### DISCUSSION TITLE
بسته شدن

### DISCUSSION THREAD

**Discussion by lucifersanthem** [MortezaBashsiz/CFScanner#166] [Q&A]
سلام خسته نباشید توی این ورژن آخر که دیروز رلیز کردین وقتی بعد از اسکن ریزالت رو انتخاب میکنم برنامه کرش میکنه کامل در کل روی هر ریزالتی میزنم این توری میشه کار خاصی باید انجام بدم؟

**Comment by goingfine** [comment]
توی بخش issues مطرح کنید با جزیات دقیق
